fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,12 +3,19 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model: function() {
     var deployment = this.modelFor('deployment');
-    return this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
+    return Ember.RSVP.hash({
```

Comment:
Oh, I commented it out because it doesn't exist in the workflow for the 'develop' branch.  It'll need to be uncommented later.